satellite/gracefulexit/common.go

```diff
@@ -22,6 +22,10 @@ type Config struct {
 	ChoreBatchSize int           `help:"size of the buffer used to batch inserts into the transfer queue." default:"500"`
 	ChoreInterval  time.Duration `help:"how often to run the transfer queue chore." releaseDefault:"30s" devDefault:"10s"`
 
-	EndpointBatchSize   int `help:"size of the buffer used to batch transfer queue reads and sends to the storage node." default:"100"`
-	EndpointMaxFailures int `help:"maximum number of transfer failures per piece." default:"3"`
+	EndpointBatchSize int `help:"size of the buffer used to batch transfer queue reads and sends to the storage node." default:"100"`
+
+	MaxFailuresPerPiece int `help:"maximum number of transfer failures per piece." default:"3"`
+	// TODO: what's the default number?
+	OverallMaxFailuresPercentage int           `help:"maximum percentage of transfer failures per node." default:"90"`
```

Comment:
This should be 10, not 90. Otherwise we allow 90% of a node's pieces to fail transfer before marking it as a failed exit. I am assuming you meant if over 10% fail, we mark as failed.